rockcraft/services/package.py

```diff
@@ -185,9 +185,6 @@ def _pack(
         datetime.datetime.now(datetime.timezone.utc).isoformat(), base_digest
     )
     rock_metadata["architecture"] = build_for
-    # TODO: add variant to rock_metadata too
```

Comment:
`TODO`s in the code were removed. What about creating issues for those that were still making sense?